sisyphus/job_path.py

```diff
@@ -239,12 +239,19 @@ def __hash__(self):
         # TODO Check how uninitialized object should behave here
         return hash((self.__dict__.get("creator"), self.__dict__.get("path")))
 
+    def _get_state_with_creator(self):
+        d = self.__dict__.copy()
+        del d["users"]
+        return d
```

Comment:
Can you add a comment describing why we need this function (for creating accurate hashes) and why we still need to delete `users`?

Reply:
The users were already deleted in the old code. Not sure why.

Will add a comment about the hashes.